common/src/python/outputs/errors.py

```diff
@@ -92,9 +93,28 @@
         "must also have a higher visit number (VISITNUM)"
     ),
     SysErrorCodes.MISSING_SUBMISSION_STATUS: (
-        "Missing submission status (MODE<form name>) variables {0}"
+        "Missing submission status (MODE<form name>) variables {0} "
         "for one or more optional forms"
     ),
+    SysErrorCodes.CLINICAL_FORM_REQUIRED_NP: (
+        "Participant must have a UDS, BDS, or MDS packet submitted before "
+        "the NP form can be submitted"
+    ),
+    SysErrorCodes.DEATH_DENOTED_ON_MLST: (
+        "DECEASED and AUTOPSY should equal 1 in the most recent Milestone form "
+        "in order for the NP form to be accepted "
+    ),
+    SysErrorCodes.DEATH_DATE_MISMATCH: (
+        "Date of death of the most recent Milestone form (DEATHMO, DEATHDY, DEATHYR) "
+        "must match the date of death on the on NP form (NPDODMO, NPDODDY, NPDODYR)"
+    ),
+    SysErrorCodes.UDS_NOT_EXIST: (

```

Comment:
This wouldn't work since both UDS_NOT_EXIST and CLINICAL_FORM_REQUIRED mapped to the same key preprocess-error